One-Electron-Property/Restrained-Electrostatic-Potential/resp_driver.py

```diff
@@ -59,6 +59,7 @@ def resp(molecules, options_list=[], intermol_constraints={}):
     # VDW surface options
     if not ('ESP' in options.keys()):
         options['ESP'] = []
+        import psi4
```

Comment:
Why have `import psi4` here? I feel like this could run into namespace errors as you added calls to psi4 below. As these are all in `if` statements we could potentially find a set of options where we miss the correct import.

Reply:
This option gives a path to a file that contains the electrostatic potential. Only if this is not provided will you need to import psi4 to calculate the ESP. That is, this RESP code can work independently of psi4 if needed.